src/components/Login/LoginContainer.tsx

```diff
@@ -149,13 +171,14 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
     }
   };
```

Comment:
small nit, for this function, do you think we should rename `fullNumber` to `fullMobileNumber`? for consistency lol. if too small a nit, can ignore me also 😂